docker/config/local_dev.env

```diff
@@ -130,6 +130,14 @@ CRASHPUBLISH_ACCESS_KEY=foo
 CRASHPUBLISH_SECRET_ACCESS_KEY=foo
 CRASHPUBLISH_QUEUE_NAME=local-dev-standard
 
-# pubsub emulator
-# ---------------
+# pubsub
+# ------
 PUBSUB_EMULATOR_HOST=pubsub:5010
+
+PUBSUB_PROJECT_ID=test
+PUBSUB_STANDARD_TOPIC_NAME=local-standard-topic
+PUBSUB_PRIORITY_TOPIC_NAME=local-priority-topic
+PUBSUB_REPROCESSING_TOPIC_NAME=local-reprocessing-topic
+PUBSUB_STANDARD_SUBSCRIPTION_NAME=local-standard-sub
+PUBSUB_PRIORITY_SUBSCRIPTION_NAME=local-priority-sub
+PUBSUB_REPROCESSING_SUBSCRIPTION_NAME=local-reprocessing-sub
```

Comment:
We need to add equivalents for topic and subscription to `docker/config/test.env`.